osu.Game/Screens/Edit/Editor.cs

```diff
@@ -456,6 +468,10 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
                     menuBar.Mode.Value = EditorScreenMode.Verify;
                     return true;
 
+                case GlobalAction.EditorTestGameplay:
+                    testGameplay();
```

Comment:
We usually click the button instead for feedback (once https://github.com/ppy/osu/issues/14132 is addressed though).

Reply:
Seems valid, will change that here.